src/commcare_cloud/environment/main.py

```diff
@@ -63,7 +64,7 @@ def check(self):
             self.fab_settings_config
             self.postgresql_config
             self.proxy_config
-        self.create_generated_yml()
+        self._create_generated_yml(save=False)
```

Comment:
If we aren't saving the generated.yml file, why is it necessary to call this?

Reply:
I think this is just a check to ensure there are no errors when generating variables.